src/interfaces/README.md

```diff
@@ -14,4 +14,6 @@ The following interfaces are defined here:
 
 * [`Init`](init.h) — used by multiprocess code to access interfaces above on startup. Added in [#10102](https://github.com/bitcoin/bitcoin/pull/10102).
 
+* [`Base`](base.h) — base interface class used by multiprocess code for bookkeeping and cleanup. Added in [#10102](https://github.com/bitcoin/bitcoin/pull/10102).
```

Comment:
I think `init.h` is actually introduced in next commit. Also document `echo.h` if it's a long-standing debug interface ?